llvm/tools/sycl-post-link/ModuleSplitter.cpp

```diff
@@ -224,71 +230,9 @@ EntryPointGroupVec groupEntryPointsByScope(const Module &M,
     // No entry points met, record this.
     EntryPointGroups.push_back({GLOBAL_SCOPE_NAME, {}});
   }
-
   return EntryPointGroups;
 }
 
-// For device global variables with the 'device_image_scope' property,
```

Comment:
This function was just moved out of the anonymous namespace into `ModuleSplitterBase` to make it callable from the `sycl-post-link.cpp`. It was also renamed to `verifyNoCrossModuleDeviceGlobalUsage` to reflect the purpose more closely.